[docs] CDCSDK: Update CDC documentation with missing sections (#21512)
yugabyte/yugabyte-db: docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md

{{< /tip >}}

## Connector compatibility

> 

Reviewer: The connector is compatible with the following versions of YugabyteDB.

| YugabyteDB | Connector |
| :--- | :--- |
| 2.14 (EA) | 1.9.5.y.3 |
| 2.16 | 1.9.5.y.24 |
| 2.18.2 | 1.9.5.y.33.2 |
| 2.20 | 1.9.5.y.220 |

{{< note title="Note" >}}

> Starting 2.20, the release naming convention has changed and we follow a scheme *major.y.minor* where

Reviewer: Starting with YugabyteDB v2.20, the naming convention for releases of the connector uses the scheme *major.y.minor*, as follows:
- major - Debezium release the connector is based on
- minor - version of YugabyteDB the connector works with
The connector is backward compatible with previous releases of YugabyteDB unless stated otherwise.

When the connector is running, the YugabyteDB server that it is connected to could become unavailable for any number of reasons. If this happens, the connector fails with an error and stops. When the server is available again, restart the connector.

> The YugabyteDB connector externally stores the last processed offset in the form of a checkpoint. After a connector restarts and connects to a server instance, the connector communicates with the server to continue streaming from that particular offset. This offset is available as long as the stream ID remains intact. Never delete a stream ID without deleting all the associated connectors with it otherwise you will lose data.

Reviewer: ```suggestion
The YugabyteDB connector externally stores the last processed offset in the form of a checkpoint. After a connector restarts and connects to a server instance, the connector communicates with the server to continue streaming from that particular offset. This offset is available as long as the stream ID remains intact. Never delete a stream ID without deleting all the associated connectors with it, otherwise you will lose data.
```

## Dropping a table part of the replication

> While the connector is running with a set of tables configured to capture the changes, if one of the tables in the set is dropped, the connector will fail with an appropriate error message indicating that the object is not found.

Reviewer: ```suggestion
While the connector is running with a set of tables configured to capture the changes, if one of the tables in the set is dropped, the connector will fail with an error message indicating that the object is not found.
```

> To avoid a connector failure or to resolve the failure, the recommended way is to follow these steps:

Reviewer: ```suggestion
To avoid or resolve a failure due to a dropped table, follow these steps:
```

> 1. Delete the connector which contains the dropped or the table to be dropped.

Reviewer: ```suggestion
1. Delete the connector that contains the table that was dropped, or that you want to drop.
```
